src/medal_app/events.cljs

```diff
@@ -30,13 +28,12 @@
 (re-frame/reg-event-db
   ::fetch-medals-success
   (fn [db [_ data]]
-    (def medalData (map (fn [d]
-      (let [{:keys [gold silver bronze]} d]
-        (assoc d :total (+ gold silver bronze))
-    )) data))
-    (-> db
-      (assoc :loading false)
-      (assoc :medals (reverse(sort-by (juxt :gold :silver) medalData))))))
+    (let [medal-data (map (fn [d]
+                       (let [{:keys [gold silver bronze]} d]
+                         (assoc d :total (+ gold silver bronze)))) data)]
```

Comment:
1. It's better to use `let` than `def` within a function.
2. `medalData` -> `medal-data` because this is typical Clojure convention. According to StackOverflow, it's called "kebab-case" or "lisp-case": https://stackoverflow.com/questions/11273282/whats-the-name-for-hyphen-separated-case